Gifski/Gifski.swift

```diff
@@ -90,20 +83,38 @@ final class Gifski {
 			return progress.isCancelled ? 0 : 1
 		}
 
+		var gifData = NSMutableData()
+
+		gifski.setWriteCallback(context: &gifData) { bufferLength, bufferPointer, context in
+			guard
+				bufferLength > 0,
+				let bufferPointer = bufferPointer
+			else {
+				return 0
+			}
+
+			let data = context!.assumingMemoryBound(to: NSMutableData.self).pointee
+			data.append(bufferPointer, length: bufferLength)
+
+			return 0
+		}
```

Comment:
@kornelski I could use an extra set of eyes on this part.

Reply:
LGTM